lib/msal-browser/package.json

```diff
@@ -78,9 +78,9 @@
     "jsdom": "^20.0.0",
     "jsdom-global": "^3.0.2",
     "rimraf": "^3.0.0",
-    "rollup": "^2.46.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-typescript2": "^0.29.0",
+    "rollup": "^3.14.0",
+    "@rollup/plugin-terser": "^0.4.0",
+    "rollup-plugin-typescript2": "^0.34.1",
```

Comment:
Did you consider switching to the [official typescript plugin](https://www.npmjs.com/package/@rollup/plugin-typescript)?

Reply:
Nice.  I didn't know that existed.  In the case of terser the original plugin was deprecated.  I'll give this a try.  Thanks!